src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -848,6 +859,15 @@ public async Task ErrorReadingModuleFile(bool breakMode)
             Assert.Empty(updates.Updates);
             AssertEx.Equal(new[] { $"{document2.Project.Id}: Error ENC1001: {string.Format(FeaturesResources.ErrorReadingFile, moduleFile.Path, expectedErrorMessage)}" }, InspectDiagnostics(emitDiagnostics));
 
+            // correct the error:
+            var correctedMvid = EmitLibrary(source2);
```

Comment:
Did you intend to use `correctedMvid` to verify something?

Reply:
No, it's not needed.